src/renderer/services/bitcoin/fees.ts

```diff
@@ -38,6 +38,7 @@ export const createFeesService = (oClient$: Client$): FeesService => {
     mergeMap(([oClient, _]) =>
       FP.pipe(
         oClient,
+        O.alt(() => O.some(new BitcoinClient({}))),
```

Comment:
Passing `nodeUrl` + `nodeApiKey` is needed here ^. Check [getFeesWithRates](https://github.com/xchainjs/xchainjs-lib/blob/master/packages/xchain-bitcoin/src/client.ts#L235) for details

Reply:
Previous comment seems to be still relevant ^.

Reply:
seems no. this comment is outdated